### DISCUSSION TITLE
Support lifetime of v6.x once v7 is stable

### DISCUSSION THREAD

**Discussion by eerhardt** [rabbitmq/rabbitmq-dotnet-client#1532] [Q&A] [answered]
What is the support policy / lifetime of the `v6.x` version of RabbitMQ.Client after `v7` is generally available as "stable"? Will `v6.x` still be maintained? How long will customers have to update to the new version before the `v6.x` is no longer supported?

cc @lukebakken

**Comment by michaelklishin** [comment, marked as answer]
@eerhardt 6.x will be maintained for some time but not for long (probably up to six months at most). 

However, this is open source software, so as long as you and other 6.x users are willing to maintain it and work on backports, this period can be longer but a small team generally cannot afford to maintain several majors for years.

**Reply by lukebakken** [reply]
There's no reason that the community can't "support" the `6.x` series as long as people would like. We'll accept bugfixes as long as they have the appropriate tests. We probably won't accept new features that aren't already in 7.x.

It would be great if the community could backport the OpenTelemetry support that is going into 7.0

**Reply by eerhardt** [reply]
Do you have any idea when 7.0 goes "stable" / "GA"?  is it sometime this year?

**Reply by lukebakken** [reply]
I'm hoping for mid-May 🤞 I will have a new alpha release soon, maybe today. The more that people test alpha releases, the sooner I get the RCs out!

**Reply by lukebakken** [reply]
@eerhardt - please give this a try!

https://www.nuget.org/packages/RabbitMQ.Client/7.0.0-alpha.5

**Reply by eerhardt** [reply]
Thanks @lukebakken. As I figured, the API breaking changes in v7 will cause us to need to rev our package. I opened https://github.com/dotnet/aspire/issues/3956 to track handling these breaking changes in .NET Aspire.

**Reply by lukebakken** [reply]
https://github.com/rabbitmq/rabbitmq-dotnet-client/releases/tag/v7.0.0-alpha.6
